### PR TITLE
📝 Create links to all other repositories

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,59 @@ or for the Home Office.
...
+
+[hocs-data 🔒️ *(deprecated)*](https://github.com/UKHomeOffice/hocs-data)
+
+[hocs-data-wcs 🔒️ *(deprecated)*](https://github.com/UKHomeOffice/hocs-data-wcs)
+
+[hocs-docs](https://github.com/UKHomeOffice/hocs-docs)
+
+[hocs-docs-converter](https://github.com/UKHomeOffice/hocs-docs-converter)
+
+[hocs-info-service *(deprecated)*](https://github.com/UKHomeOffice/hocs-info-service)
```
Comment (SWhitlam) [inline README.md:35]: To avoid confusion should we mark this as "To be deprecated"?

Comment (LiamMacP) [reply README.md]: Is this to be deprecated - when we discussed some referential data makes sense in it.

Comment (nickmcmahon01) [reply README.md]: you'd build a reference data service differently to hocs-info-service, shall I just remove the all brackets?

Comment (LiamMacP) [reply README.md]: Just remove this one from hocs-info-service till we actually deprecate.

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**